### ISSUE TITLE
`quarkus.container-image.image` not working as expected on OpenShift, wrong `imagestream.yaml`

### ISSUE THREAD

**Issue by pjgg** [quarkiverse/quarkus-helm#100]
Extension version: `0.0.6`

When I deploy an application through `Quarkus-helm` extension over OpenShift with the following configuration

Application.properties
```
quarkus.openshift.route.expose=true
quarkus.container-image.image=quay.io/quarkusqeteam/ts-quarkus-helm-simple-app:latest
quarkus.helm.name=my-chart
quarkus.container-image.builder=docker
```

the generated `imagestream.yaml` is poing to `docker.io` instead of `quay.io`

Workaround:

Application.properties
```
quarkus.openshift.route.expose=true
quarkus.helm.name=my-chart
quarkus.container-image.builder=docker

quarkus.container-image.registry=quay.io
quarkus.container-image.group=quarkusqeteam
quarkus.container-image.name=minimum-quarkus-helm
quarkus.container-image.tag=v1
```

**Comment by Sgitario** [comment]
I can reproduce the issue, but it's a Quarkus issue, not a Quarkus Helm issue. 
For the record, what this extension is doing is copying the generated resources by the Quarkus Kubernetes/Openshift extensions into the folder `helm` and mapping some keys into the `values.yaml`. 
Please, report the issue in Quarkus. I'm closing this.

**Comment by Sgitario** [comment]
This issue should now be fixed using Quarkus 2.13.2.Final.
